go.work

```diff
@@ -0,0 +1,3 @@
+go 1.20
```

Comment:
In the grpc-go repo, we don't use the go.work file and we leave it to the user to use one if they want to, and add it to their .gitignore.

Why is this file required?